x-pack/plugins/observability/public/components/app/section/index.tsx

```diff
@@ -4,21 +4,23 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { EuiAccordion, EuiLink, EuiPanel, EuiSpacer, EuiText, EuiTitle } from '@elastic/eui';
-import { i18n } from '@kbn/i18n';
 import React from 'react';
 import { ErrorPanel } from './error_panel';
 import { usePluginContext } from '../../../hooks/use_plugin_context';
 
+interface AppLink {
+  label: string;
+  href?: string;
+}
+
 interface Props {
   title: string;
   hasError: boolean;
   children: React.ReactNode;
-  minHeight?: number;
-  appLink?: string;
-  appLinkName?: string;
+  appLink?: AppLink;
```

Comment:
Shouldn't this be required?

Reply:
Btw any reason not to inline `AppLink` ?

Reply:
> Shouldn't this be required?

This is optional because it comes from the call of `fetchData`, and we show the section with the loading while waiting for the response, otherwise, we'd have to hide the entire <Section> until the response comes up.